brianlong/www.validators.app
# 180409120_filter_by_private_validators

#### What's this PR do?
- Add ability to filter by private validators + show or hide private validators from set.

#### How should this be manually tested?
- Go to http://localhost:3000/data-centers?locale=en&network=mainnet
- See the data center whith at least one private validator
- Check new buttons: 'Filter by private'  if it shows only private and 'Show/Hide private' if it include/exclude private validator from the set.

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/180409120

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
